src/api/telemetry/TelemetryCollection.js

```diff
@@ -62,8 +75,8 @@ export default class TelemetryCollection extends EventEmitter {
             this._error(LOADED_ERROR);
         }
 
-        this._setTimeSystem(this.openmct.time.timeSystem());
-        this.lastBounds = this.openmct.time.bounds();
+        this._setTimeSystem(this.options.timeContext.timeSystem());
```

Comment:
This will error out if options === {} right?

Reply:
it should be ok, as few lines above we call `this.openmct.telemetry.standardizeRequestOptions(options);` that adds a `timeContext` if one wasn't supplied.

Reply:
Please see comment above about this in `standardizeRequestOptions`